### PR TITLE
Api authorization

### PR DESCRIPTION
**Pull request by easton-morris**

### CODE REVIEW

**Comment by easton-morris** [comment]
image

**Comment by easton-morris** [comment]
image

**Comment by easton-morris** [comment]
image

File: api-authorization/server/authorization-middleware.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const jwt = require('jsonwebtoken'); // eslint-disable-line
+const ClientError = require('./client-error'); // eslint-disable-line
+
+function authorizationMiddleware(req, res, next) {
+  /* your code here */
+  const xToken = Object.getOwnPropertyDescriptor(req.headers, 'x-access-token');
```
Comment (scbowler) [inline api-authorization/server/authorization-middleware.js:6]: This is a bit overkill, you can use the `req.headers` object like any other object:

```JAVASCRIPT
cont xToken = req.headers['x-access-token'];
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just one update needed. Let me know if you have any questions.

**Review by scbowler — APPROVED** [review]
Great job with the update, looks good! 👍

**Pull request merged**